packages/arb-token-bridge-ui/src/components/TransferPanel/Routes/Routes.tsx

```diff
@@ -0,0 +1,110 @@
+import { PropsWithChildren, useEffect } from 'react'
+import { useNetworks } from '../../../hooks/useNetworks'
+import { useNetworksRelationship } from '../../../hooks/useNetworksRelationship'
+import { useIsCctpTransfer } from '../hooks/useIsCctpTransfer'
+import { useIsOftV2Transfer } from '../hooks/useIsOftV2Transfer'
+import { ArbitrumCanonicalRoute } from './ArbitrumCanonicalRoute'
+import { CctpRoute } from './CctpRoute'
+import { OftV2Route } from './OftV2Route'
+import React from 'react'
+import { useRouteStore } from '../hooks/useRouteStore'
+import { useArbQueryParams } from '../../../hooks/useArbQueryParams'
+import { useSelectedToken } from '../../../hooks/useSelectedToken'
+
+function Wrapper({ children }: PropsWithChildren) {
+  return <div className="mb-2 flex flex-col gap-2">{children}</div>
+}
+
+export function useSetSelectedRoute() {
```

Comment:
Should this be it's own file? Also we must rename it to `useDefaultSelectedRoute()` or something to avoid confusion with `setSelectedRoutes` function.

Reply:
Not for now, it's tightly coupled to Routes component, so it make more sense to have it here